### PR TITLE
Close if not a review issue

### PR DESCRIPTION
**Pull request by arfon**
This PR implements a change to the Whedon behaviour to make sure that if an issue is opened that isn't a `pre-review` or `review` issue, Whedon will auto-close it and redirect the author of the issue.

### CODE REVIEW

**Comment by arfon** [comment]
This is what it looks like in testing:

Screen Shot 2021-08-06 at 11 56 48 AM

File: responses/close.erb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+:wave: This repository is only for review issues (`pre-review` and `review`) that have been created by our editorial infrastructure, and this issue appears not to be one of these.
+
+As such, this issue will be closed. If you're opening an issue as feedback for a review, please open a new issue instead on the repository for the submission you are reviewing.
```
Comment (danielskatz) [inline responses/close.erb:3]:
```suggestion
As such, this issue will be closed. If you're opening an issue as part of a review, please open a new issue instead in the software repository associated with the submission that you are reviewing.
```

**Comment by danielskatz** [comment]
Other than my suggested wording change, this mostly looks good to me.
(I'm also a little confused why we can't have reminders in pre-review issues.)

**Comment by arfon** [comment]
> (I'm also a little confused why we can't have reminders in pre-review issues.)

I don't actually recall. Lemme take a look into that separately :-)

**Pull request merged**